app/src/main/res/values-ar/strings.xml

```diff
@@ -20,6 +20,8 @@
     <string name="unpin_conversation">ازالة التثبيت</string>
     <string name="forward_message">اعادة ارسال</string>
     <string name="compress_error">غير قادر على ضغط الصورة إلى الحجم المحدد</string>
+    <!-- Attachments-->
+    <string name="duplicate_item_warning">Duplicate item was not included</string>
```

Comment:
when is this string used at all? I can add the same file multiple times

Reply:
shouldn't be happening, it'll also work for contacts.
Are you choosing a file or media?

Reply:
It happens with images because the Uri changed after compression, working on it.

Reply:
added an image multiple times